server/models/booking.py

```
@@ -51,12 +50,13 @@ class Booking(db.Model):
     service = db.relationship("Service", back_populates="bookings")
 
     # --- Data Validations ---
-    @validates("booking_date")
-    def validate_booking_date(self, key, booking_date_value):
-        """Ensure booking_date is not in the past."""
-        if booking_date_value < date.today():
-            raise ValueError("Booking date cannot be in the past.")
-        return booking_date_value
+    @validates("start_time")
+    def validate_start_time(self, key, start_time_value):
```

Ruff (0.13.3)

Unused method argument: `key`

(ARG002)